modules/account-lib/test/unit/coin/dot/transaction.ts

```diff
@@ -51,4 +60,110 @@ describe('Dot Transaction', () => {
       should.deepEqual(tx.transactionSize(), DotResources.rawTx.transfer.unsigned.length / 2);
     });
   });
+
+  describe('Dot Explain Transaction', () => {
```

Comment:
Add failure cases.

Reply:
We can never reach to a failure with this as the transaction building would fail for any sort of invalid transaction because of the other internal checks.